ads/google/a4a/utils.js

```diff
@@ -143,48 +143,49 @@ export function googleAdUrl(
   // TODO: Maybe add checks in case these promises fail.
   /** @const {!Promise<string>} */
   const referrerPromise = viewerForDoc(a4a.getAmpDoc()).getReferrerUrl();
-  return getAdCid(a4a).then(clientId => referrerPromise.then(referrer => {
-    const adElement = a4a.element;
-    window['ampAdGoogleIfiCounter'] = window['ampAdGoogleIfiCounter'] || 1;
-    const slotNumber = window['ampAdGoogleIfiCounter']++;
-    const win = a4a.win;
-    const documentInfo = documentInfoForDoc(adElement);
+  return getAdCidHelper(a4a.getAmpDoc(), 'AMP_ECID_GOOGLE').then(
```

Comment:
please add '_ga' as the 3rd param of the call, since we're migrating to GA cookie name. see #9591